tools/openms/AccurateMassSearch.xml

```diff
@@ -14,6 +14,7 @@
   <command detect_errors="exit_code"><![CDATA[@QUOTE_FOO@
 @EXT_FOO@
 #import re
+#import os.path
```

Comment:
just import os?

Reply:
If you like I can change this. Since only functionality from os.path is needed I wanted to be specific.